### DISCUSSION TITLE
User maps

### DISCUSSION THREAD

**Discussion by ph2708** [netpanzer/netpanzer#76] [Maps]
I would suggest adding those maps that were created by former game maintainers.

If possible, it would be really cool for the game to automatically download maps that were not found when starting a match on the server.

[maps.zip](https://github.com/netpanzer/netpanzer/files/13859406/maps.zip)

**Comment by andy5995** [comment]
> I would suggest adding those maps that were created by former game maintainers.

They'd probably need to be reviewed one at a time. Can you open a ticket or PR to suggest one?

> 
> If possible, it would be really cool for the game to automatically download maps that were not found when starting a match on the server.

Yes, could you please open a ticket for that?
